server_cmds.go

```diff
@@ -15,129 +15,149 @@ type ServerMethods struct {
 }
 
 // Instance represents basic information for a TeamSpeak 3 instance.
+// instanceinfo
```

Comment:
Random addition?

Reply:
those are actually just mental notes.
I did those to better visually distinguish which command is actually used.